live-build/misc/ansible-roles/appliance-build.virtualization-common/tasks/main.yml

```diff
@@ -45,3 +45,34 @@
       domain=
       dns1=
       dns2=
+
+- name: Add the CLI user with home directory
```

Comment:
I think all of the changes to this Ansible role should actually live in the app-gate. Creating the CLI user should live in the post-install hook for the app-gate package, and the sshd configuration should live in the delphix-config role.

Why did you choose this approach vs. what I'm suggesting?

Reply:
I thought of this location as part of the appliance preparation, but the post-install hook as you suggest might be a better place (more robust if we ever install the package without going through this role).